popup/js/popup.js

```diff
@@ -1,6 +1,55 @@
+//Autofill Logic
+var tex=document.getElementById("query");
```

Comment:
:P The variable names are a bit fuzzy, IMO. Couldn't you just use the full name as in `suggestion` and `text`?

Reply:
Right!
Done 👍